### PR TITLE
Feature/ng 10

### PR DESCRIPTION
**Pull request by emclaug2**
#### Changes proposed in this Pull Request:
- Update to angular 10
- http://stackblitz.com/github/pxblue/ionic-design-patterns/tree/feature/ng-10

### CODE REVIEW

File: angular.json [hunk 1]
```diff
@@ -1,41 +1,36 @@
...
+              "src/favicon.ico",
+              "src/assets"
             ],
             "styles": [
               "src/styles.scss",
-              "./node_modules/@pxblue/angular-themes/theme.scss",
-              "./node_modules/placeholder-loading/src/scss/placeholder-loading.scss",
-              "./node_modules/typeface-open-sans"
+              "./node_modules/typeface-open-sans",
+              "./node_modules/@pxblue/angular-themes/theme.scss"
```
Comment (joebochill) [inline angular.json:33]: where did the placeholder loading styles go for the loading states example?

File: angular.json [hunk 2]
```diff
@@ -96,59 +89,45 @@
...
+            ],
+            "scripts": []
           }
         },
-        "ionic-cordova-build": {
-          "builder": "@ionic/angular-toolkit:cordova-build",
+        "lint": {
+          "builder": "@angular-devkit/build-angular:tslint",
```
Comment (joebochill) [inline angular.json:105]: Is this ts lint stuff supposed to be here if we are using eslint? Is this just boilerplate from angular cli?

**Review by joebochill — COMMENTED** [review]
Changes look reasonable...can't run it on my end just yet.

File: angular.json (hunk 1 above)
Comment (emclaug2) [reply angular.json]: I must have forgotten to migrate these styles over; I bet the angular-design-patterns repo has the same problem.

File: angular.json (hunk 2 above)
Comment (emclaug2) [reply angular.json]: Yep you're correct, the lint config is now removed since it's not utilized.

**Review by joebochill — CHANGES_REQUESTED** [review]
image

**Comment by emclaug2** [comment]
Oops, I forgot to test this out on mobile.  I've added the missing mobile builder to package.json & mobile config to the angular.json file.

This should be working now, tested on android & browser.

**Review by joebochill — APPROVED** [review]

**Pull request merged**